gallery/src/components/NxDropdown/NxDropdownPage.tsx

```diff
@@ -132,18 +132,18 @@ const NxDropdownPage = () =>
               </td>
             </tr>
             <tr className="nx-table-row">
-              <td className="nx-cell">HTML div Attributes</td>
+              <td className="nx-cell">HTML <code className="nx-code">&lt;div&gt;</code> Attributes</td>
               <td className="nx-cell">
                 <a target="_blank"
```

Comment:
NxTextLink

Reply:
> NxTextLink

Removed here and everywhere else I could find it: https://github.com/sonatype/sonatype-react-shared-components/pull/397/commits/b44716991bafacdbfadd26200cce784c7bc998dc

Reply:
> Removed here and everywhere else I could find it (in the docs tables): b447169

Missed `NxColorPicker`, `NxCodeSnippet`

Reply:
> > Removed here and everywhere else I could find it (in the docs tables): [b447169](https://github.com/sonatype/sonatype-react-shared-components/commit/b44716991bafacdbfadd26200cce784c7bc998dc)
> 
> Missed `NxColorPicker`, `NxCodeSnippet`

https://github.com/sonatype/sonatype-react-shared-components/pull/397/commits/cd46960181c853c9f196ab97a7e377e23d72de4f

Reply:
> > Missed NxColorPicker, NxCodeSnippet
> 
> cd46960

That wasn't the one I meant in `NxCodeSnippet`.  The one relevant to this PR is still there.

Reply:
> > > Missed NxColorPicker, NxCodeSnippet
> > 
> > 
> > [cd46960](https://github.com/sonatype/sonatype-react-shared-components/commit/cd46960181c853c9f196ab97a7e377e23d72de4f)
> 
> That wasn't the one I meant in `NxCodeSnippet`. The one relevant to this PR is still there.

https://github.com/sonatype/sonatype-react-shared-components/pull/397/commits/d104aca68e7e083b89665246c20869978c05cce4